docs/misc/sprite-animator.mdx

```diff
@@ -15,7 +15,71 @@ sourcecode: src/core/SpriteAnimator.tsx
 
 ```tsx
 
-/** Common properties that can be safely applied to both sprite and instance components */
+/**
+ * Source size information for a sprite frame
+ */
+interface SourceSize {
```

Comment:
Not prescriptive, but we generally prefer using `type` over `interface` in the project

Reply:
done in latest commit